### ISSUE TITLE
AGOL Account Clean-Up

### ISSUE THREAD

**Issue by jaime-mckeown** [cityofaustin/atd-geospatial#104]
Ben from CTM sent out a list of AGOL accounts that have not logged in for 6 months or 1 year. I will need to notify the users so they can login to keep their accounts from being demoted or deleted. Account changes occur on 7/19/2019

G:\ATD\ATD_GIS\01_ADMIN\ArcGIS_Online\Account_Clean_up

**Comment by jaime-mckeown** [comment]
Notified users via email of their AGOL account status. Received a few emails from users that I answered.

**Comment by jaime-mckeown** [comment]
- I asked Ben how he determines what department people are in. He mentioned that the Bio section for user profiles is how he searches. I noticed that many user profile Bio's were incorrect or not filled out. Diana told me that is was something she would do to keep the user profiles up-to-date.
- I then got a department list from Scott so I had an accurate list to go by and went down the list. I found many users in our department that Ben missed is his notification. I updated all user profiles and need to send another notification letting them know that they need to login.
- I want to mention this problem at the next AGOL Subcommittee Meeting as there should be a better way to keep the user profile Bio's up-to-date then SPOCs having to do it.
